src/core/a-entity.js

```diff
@@ -127,6 +129,28 @@ var proto = Object.create(ANode.prototype, {
     }
   },
 
+  getObject3D: {
```

Comment:
i'm getting a bit confused on `getObject3D` vs `setObject3D` since they both do setting. i would probably get it if i read it more deeply, but will have to think about something that is clearer from the get-go.